client/assets/stylesheets/_wp-components-overrides.scss

```diff
@@ -1,8 +1,3 @@
-/* @wordpress/components Form toggles use the color scheme's background color */
-.components-form-toggle.is-checked .components-form-toggle__track {
-	background-color: var(--color-primary);
-}
-
```

Comment:
The fun thing is that we've added this code in https://github.com/Automattic/wp-calypso/pull/48515 to add the feature which we are fixing here by removing it.

Any idea how it happensed? Did we change how `--color-primary` works over the years?

Reply:
It looks as though `--color-primary` is not the ideal choice. For some themes it gives the expected color for the toggle but for some themes it doesn't. 

Having tested with most of the available themes, removing this option gives the expected results. Maybe it was implemented and then tested with a theme where the primary color was good for the toggle.

Reply:
> Maybe it was implemented and then tested with a theme where the primary color was good for the toggle.

Thanks, it may be that. 👍